coaster-blas/src/frameworks/cuda/helper.rs

```diff
@@ -250,32 +251,56 @@ macro_rules! iblas_gemm_for_cuda {
             beta: &SharedTensor<$t>,
             c: &mut SharedTensor<$t>,
         ) -> Result<(), ::coaster::error::Error> {
-            let c_desc = c.desc().clone();
-            let alpha_mem = read!(alpha, self);
-            let beta_mem = read!(beta, self);
-            let a_mem = read!(a, self);
-            let b_mem = read!(b, self);
-            let c_mem = write_only!(c, self);
+            use Transpose as T;
```

Comment:
```suggestion
            use Transpose::{NoTrans, ConjTrans, Trans};
```
is a bit more idiomatic.